### ISSUE TITLE
Auto Update not working

### ISSUE THREAD

**Issue by Lovecore** [NecronomiconCoding/Pokemon-Go-Rocket-API#48]
Auto-update will not 'auto aupdate'. I have to manually pull down the project.

**Comment by linkz-db** [comment]
I personally use git pull to update newest version. I think it's the best solution for now till he fixes it.

**Comment by xDiglett** [comment]
Yes, it seems that right now it's not implemented yet. It will only show a notice to let you know.
